conf/app.routes

```diff
@@ -13,23 +13,27 @@ POST       /apply-for-help-to-save/apply                    @uk.gov.hmrc.helptos
 
 
 GET        /select-email                                    @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.getSelectEmailPage
-GET        /access-account                                  @uk.gov.hmrc.helptosavefrontend.controllers.AccessAccountController.accessAccount
+GET        /no-account                                      @uk.gov.hmrc.helptosavefrontend.controllers.AccessAccountController.accessAccount
```

Comment:
we may want to keep this endpoint and create a new endpoint for` /no-account` - this endpoint is the one for when a user clicks `Access Your Account` - if they don't have an account they should redirect to `/no-account`

Reply:
(I can sort this one out)